.github/workflows/workflow.yml

```diff
@@ -0,0 +1,28 @@
+name: workflow
+on: [pull_request]
```

Comment:
Naming suggestions welcome. Also, should we run this on events other than pull requests? Note that pull request includes both the initial PR as well as any future pushes to the PR branch, meaning the checks will re-run on every update to the PR, not just on the initial creation.

Reply:
maybe just `CI` for the name? I also can't think of a better name right now.

Re: workflow triggers, I'd also add `push`